docs/dasharo-tools-suite/documentation.md

```diff
@@ -395,10 +394,9 @@ Please consider the following options depending on your situation:
 
 DTS can be used to update Dasharo firmware. To achieve this, boot it on platform
 with flashed Dasharo and choose option number `2`. You may see additional
-information about available updates if you are not [Dasharo Entry
-Subscription](https://docs.dasharo.com/ways-you-can-help-us/#become-a-dasharo-entry-subscription-subscriber)
+information about available updates if you are not [Dasharo Pro Package](https://docs.dasharo.com/ways-you-can-help-us/#become-a-dasharo-entry-subscription-subscriber)
```

Comment:
And here DES is left here in the link

Reply:
https://github.com/Dasharo/docs/compare/ab89e913eb96df9b51fbf8927a4e330b3159476b..5b987bfac03c93fceea6394945f7936e91e20fe2#diff-8cfee4c026a23b6eea51be5305d2c9db7d12139283fcec2ee3522ffffc0dc026R397

Updated all the links which were left out including links to the shop.3mdeb